src/components/ButtonPanel.js

```diff
@@ -1,40 +1,45 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-function ButtonPanel() {
+function ButtonPanel({ clickHandler }) {
```

Comment:
- [ ] It will be great to have a `handleClick` function in this component. This is a requirement for this milestone to be approved. Example: 

`handleClick = (buttonName) => {
  this.props.clickHandler(buttonName);
};
`
![implementHandleClick](https://user-images.githubusercontent.com/32361963/129005615-739e1692-7e9a-43a3-baf7-53905fe122e7.png)

Reply:
I guess I am doing as requested, I implemented the handleClick function on the App class component and I am returning clickHandler as a prop for buttonPanel component.